### PR TITLE
l10n into ja-JP

### PR DESCRIPTION
**Pull request by luixxiul**
Please make sure that translation has not been reviewed yet. This is a test localization.
As it was not tested locally on Brave, there may be some strings to be replaced due to inconsistency.

CC: @aekeus 

Regarding #1477

### CODE REVIEW

File: app/extensions/brave/locales/ja-JP/app.properties [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+removeFromReadingList=リーディングリストから削除
+forward=進む
+forwardButton.title=次に進む
+back=戻る
+backButton.title=前に戻る
+ok=OK
+cancel=キャンセルする
+submit=送信
+settings=設定
+aboutPages=About pages
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/app.properties]: Not translated on Google Chrome

File: app/extensions/brave/locales/ja-JP/app.properties [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+listOfAboutPages=List of about pages
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/app.properties]: Not translated on Google Chrome

File: app/extensions/brave/locales/ja-JP/menu.properties [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+copyLinkAddress=リンクのアドレスをコピー
+copyEmailAddress=メールアドレスをコピー
+saveLinkAs=名前を付けてリンクを保存
+saveImage=画像を保存
+openImageInNewTab=新しいタブで画像を開く
+copyImageAddress=画像のアドレスをコピー
+viewPageSource=ページのソースを表示
+addToReadingList=リーディングリストに追加
+quit=終了
+about=About
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/menu.properties]: I cannot find where this is

File: app/extensions/brave/locales/ja-JP/menu.properties [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+inspectElement=要素を検証
+downloadsManager=ダウンロードマネージャー
+zoom=ズーム
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/menu.properties]: What is difference between this and "zoom in/out"?

File: app/extensions/brave/locales/ja-JP/preferences.properties [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+hint1=Braveは自動アップデートを行いますが、必要に応じて、メニューからアップデートを確認することができます。
+hint2=ファイルメニューでは新しいセッションタブを作成できます。セッションタブは普通のタブと似ていますが、別のユーザープロフィールで動きます。それによって、同じサイトに複数回ログインすることができます。
+sendUsFeedback=フィードバックを送信
+loveToHear=ぜひお聞かせください。
+startsWith=起動時
+startsWithOptionLastTime=前回開いていたウィンドウもしくはタブを表示
+startsWithOptionHomePage=ホームページを表示
+startsWithOptionNewTabPage=新しいタブページを表示
+myHomepage=ホームページ
+undefined
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/preferences.properties]: Bug?

File: app/locale.js [hunk 6]
```diff
@@ -158,8 +158,9 @@ const DEFAULT_LANGUAGE = 'en-US'
 
 const availableLanguages = [
   'en-US',
-  'pr-BR',
```
Comment (luixxiul) [inline app/locale.js]: Bug?

File: app/extensions/brave/locales/ja-JP/app.properties [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+addBookmarkButton.title=このページをブックマークに追加します
+removeBookmark=ブックマークを削除
+removeBookmarkButton.title=ブックマークを削除
+addToReadingList=リーディングリストに追加
+removeFromReadingList=リーディングリストから削除
+forward=進む
+forwardButton.title=次に進む
+back=戻る
+backButton.title=前に戻る
+ok=OK
```
Comment (luixxiul) [inline app/extensions/brave/locales/ja-JP/app.properties]: What is the context?

**Comment by aekeus** [comment]
Hi @luixxiul. Thanks for the pull request. We are managing our translations through transfex.com.

**Comment by luixxiul** [comment]
> Hi @luixxiul. Thanks for the pull request. We are managing our translations through transfex.com.
> 
> Please go to https://www.transifex.com/brave/brave-laptop/ sign up and I will add you as a translator / reviewer.

I was already added as translator, and translation was done on transifex.com.
I just want to test these files before you set up auto-downloading translation files

**Comment by aekeus** [comment]
Ok thanks for the info. There are a few changes that will be added to the next version required to support arbitrary language additions. When they land in master I will update this pull request. Thanks.

**Comment by luixxiul** [comment]
> Ok thanks for the info. There are a few changes that will be added to the next version required to support arbitrary language additions. When they land in master I will update this pull request. Thanks.

OK I see. I'm waiting for it :-)

**Comment by luixxiul** [comment]
@aekeus if possible would you please build Brave which include this translation, and give it to me at first? It helps me a lot, as I cannot build it on my environment. Thanks in advance

**Pull request closed without merging**